transport/tchannel/channel_outbound.go

```diff
@@ -74,6 +74,12 @@ type ChannelOutbound struct {
 	once *lifecycle.Once
 }
 
+// TransportName is the transport name that will be set on `transport.Request`
+// struct.
+func (o *ChannelOutbound) TransportName() string {
```

Comment:
This is a bug in tchannel transport for outbound, the transport name in the transport request is empty. The correct value is required to determine the correct propagation format and carrier.